crates/host-bench/src/lib.rs

```diff
@@ -482,50 +489,65 @@ mod powdr {
     /// - `elf`: The original ELF file, used to detect the basic blocks.
     /// - `stdin`: The standard input to the program, used for PGO data generation to choose which basic blocks to accelerate.
     pub fn apc(
-        exe: VmExe<F>,
-        vm_config: SdkVmConfig,
+        original_program: OriginalCompiledProgram<P>,
         elf: &[u8],
         apc: usize,
         apc_skip: usize,
         stdin: StdIn,
     ) -> CompiledProgram<P> {
-        let og = OriginalCompiledProgram { exe: exe.clone(), sdk_vm_config: vm_config.clone() };
-
-        let pgo_data = pgo(og, stdin.clone()).unwrap();
-
-        let bus_map =
-            BusMap::openvm_base().with_sha(7).with_bus_type(8, BusType::TupleRangeChecker);
+        let pgo_data = pgo(original_program.clone(), stdin.clone()).unwrap();
 
         let powdr_config = PowdrConfig::new(apc as u64, apc_skip as u64)
-            .with_bus_map(bus_map)
-            .with_degree_bound(powdr_constraint_solver::inliner::DegreeBound {
-                identities: 2,
-                bus_interactions: 2,
-            });
+            .with_degree_bound(powdr_openvm::DegreeBound { identities: 2, bus_interactions: 2 });
```

Comment:
```suggestion
            .with_degree_bound(powdr_openvm::DegreeBound { identities: 3, bus_interactions: 2 });
```
I *think* this should now work since we separated these bounds internally in the inliner.